File: Python/Estructuras de control/Estructuras condicionales/OrdenCreciente.py

# . Leer tres números del teclado y deducir si se han introducido en orden creciente.

#Entradas
num1 = int(input("Ingrese el primer número: "))
num2 = int(input("Ingrese el segundo número: "))
num3 = int(input("Ingrese el tercer número: "))

#Proceso
if num1 < num2 and num2 < num3:
    print("Los números están en orden creciente.")
else:
    if num1 > num2 and num2 > num3:
        print("Los números están en orden decreciente.")
    else:
        print("Los números no están en orden creciente ni decreciente.")

File: C++/Estructuras de control/Estructuras condicionales/OrdenCreciente.cpp

// . Leer tres números del teclado y deducir si se han introducido en orden creciente.

#include <iostream>
using namespace std;

int main() {
    // Entradas
    int num1, num2, num3;
    cout << "Ingrese el primer número: ";
    cin >> num1;
    cout << "Ingrese el segundo número: ";
    cin >> num2;
    cout << "Ingrese el tercer número: ";
    cin >> num3;

    // Proceso
    if (num1 < num2 && num2 < num3) {
        cout << "Los números están en orden creciente." << endl;
    } else if (num1 > num2 && num2 > num3) {
        cout << "Los números están en orden decreciente." << endl;
    } else {
        cout << "Los números no están en orden creciente ni decreciente." << endl;
    }

    return 0;
}